File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.py
def solution(s):
    answer = ''
    s_len = len(s)
    if( s_len % 2 == 0):
        middle = s_len // 2
        answer = s[middle - 1: middle+1]
    else:
        middle = s_len // 2
        answer = s[middle]
    return answer

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    if(s.length % 2 == 0)  {
        let middle = Math.floor(s.length / 2);
        answer = s.slice(middle - 1, middle + 1);
    } else {
        let middle = Math.floor(s.length / 2);
        answer = s[middle];
    }
    return answer;
}